### ISSUE TITLE
[Savage Worlds - Tabbed] Fighting starts as trained skill

### ISSUE THREAD

**Issue by djrenren** [Roll20/roll20-character-sheets#6257]
By default characters in Savage Worlds (Adventure Edition) should be untrained in fighting so their default fighting stat should be 1d4 - 2. Not a flat 1d4.

I get that we can't eliminate the field by default because it's used to calculate Parry, but hopefully it shouldn't be too hard to default the field to have the minus 2.

**Comment by finderski** [comment]
Not difficult, but since it's a rare instance that a character doesn't take at least a d4 in Fighting this was done to save the majority of people the hassle of having to delete a -2 while putting that burden on the few characters that don't take any Fighting at all.

Hope that makes sense.

**Comment by djrenren** [comment]
Yeah it makes sense and I see how it could save hassle, but I'd like to offer some pushback, if that's okay. When I'm GM'ing with new players I find it really nice to point them to your character sheet as a structured way to build their characters. The past game I hosted I had one player point this out to me and the other 2 players assumed they had just missed something in the rules and that they should start with Fighting 1d4. This is doubly confusing now that in Adventure Edition some skills really do start with 1d4.

Anyway, your call, just sharing what caused me to open the issue. Thanks for your awesome work on this! 😄

**Comment by finderski** [comment]
With the Official Savage Worlds character sheet, this is now an option. In Sheet settings you set it so Fighting starts a unskilled with the -2. You can also set that on each sheet individually, if you don't set it as a sheet setting when creating/modifying your game.
